### DISCUSSION TITLE
#129  match 20.3% while difference says much more ?

### DISCUSSION THREAD

**Discussion by gcyrillus** [cssbattle/forum#316] [Q&A] [answered]
Hello the world :)
There might be something i am missing but do not know what,  about #129  https://cssbattle.dev/play/129
I might be missing a color that i do not see myself, but i used them all.

This code  `<style>* *{background:#6292cf;filter:drop-shadow(40px 0#2e3b9f)drop-shadow(40px 0#515dbd);margin:0}:before,:after{content:'';position:fixed;width:100;height:80;border-radius:0 41%41%0/50%;background:#060f55;top:80;left:110}:before{scale:1-1;top:140`  seems to be 100% match but gives only 20.3%  

Browser used : Chrome Version 106.0.5249.91 (Build officiel) (64 bits)

![cssB](https://user-images.githubusercontent.com/1832331/193906186-f927856e-b8ca-4ecd-93d5-d55f87d359e5.jpg)

Else, would it be a bug ?

**Comment by aarthvaen** [comment, marked as answer]
Your background color is incorrect, you need `#6592CF` instead of `#6292CF`. You will then have to tweak your border-radius values since you'll be stuck with your solution at 99.9% match. 😄

**Comment by gcyrillus** [comment]
Oh thanks you, i do not only have troubles with colors, but also with reading.

**Comment by johanberonius** [comment]
Something is different with target #129, I wrote this and it looks like 100%, even without any anti aliasing mismatch. Copy-pasted the color codes. Tested in Firefox, Chrome and Safari. Scores only 80.1% match. Can you see any reason?
```
<p><p><p><p><p><p><style>*{background:#6592CF;margin:49.5 55;direction:rtl}p{width:100;height:80;margin:-19.5 0;border-radius:0 40px 40px 0;background:#515DBD;+p+p{background:#2E3B9F;translate:-40px -121px;+p+p{background:#060F55;translate:-80px -242px

**Reply by johanberonius** [reply]
No worries, rewrote even shorter. Looks the same but got 100% match.
```
<p><style>*{background:#6592CF;margin:40 55}p{width:100;height:80;margin:0;border-radius:0 40px 40px 0;background:#060F55;box-shadow:0 60px#060F55,40px 0#2E3B9F,40px 60px#2E3B9F,80px 0#515DBD,80px 60px#515DBD
